.github/workflows/update-design-review-issues.yml

```diff
@@ -0,0 +1,18 @@
+name: Update Design Review Issues
+
+on:
+  schedule:
+    # At minute 1 (every hour).
+    - cron: '1 * * * *'
```

Comment:
We don't need to run this every hour, but the cadence will affect how long is the delay to close/pin from the most recently ended session.

Reply:
One hour seems okay. Less frequently is also okay. Does the GitHub API rate limit us in any way?

Alternative suggestion: Since we already know when the meetings are happening, can we schedule this job to happen a few minutes after each meeting ends instead of every hour? This way, we can run the job once a week instead of 168 times a week.

Reply:
The script executes at least one Github request, and at the most 4 + N where N is the number of stale issues (which should be 1). So we shouldn't get close to Github's rate limiting, which is [1000 per hour in an action.](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting).

We only need to run on Wednesday, so now it runs 24 times per week instead of 168. I could alternatively configure 4 specific run times, once per possible rotation time, but I figured it's simpler to run every hour instead of keep that config in sync.